cmd/node-joiner/main.go

```diff
@@ -23,22 +16,27 @@ func main() {
 			if err != nil {
 				return err
 			}
-			return nodejoiner.NewAddNodesCommand(wd, kubeConfig)
+			dir, err := cmd.Flags().GetString("dir")
+			if err != nil {
+				return err
+			}
+			return nodejoiner.NewAddNodesCommand(dir, kubeConfig)
 		},
 	}
 
 	nodesMonitorCmd := &cobra.Command{
 		Use:   "monitor-add-nodes",
 		Short: "Monitors the configured nodes while they are joining an existing cluster",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return nodejoiner.NewMonitorAddNodesCommand(wd)
+			return nodejoiner.NewMonitorAddNodesCommand("")
 		},
 	}
 
 	rootCmd := &cobra.Command{
 		Use: "node-joiner",
 	}
 	rootCmd.PersistentFlags().String("kubeconfig", "", "Path to the kubeconfig file.")
+	rootCmd.PersistentFlags().String("dir", ".", "assets directory")
```

Comment:
I think we need to support --log-level flag either in one of the existing PRs or a follow-up PR.

Reply:
Good catch @rwsu , will address it in https://github.com/openshift/installer/pull/8093

Reply:
Is this new command will loom something line `./openshift-install agent add-nodes`? Should it be then placed in `cmd/openshift-install.node-joiner` or something like

Reply:
it's a complete separate binary